samples/devfiles/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Sample copied from https://github.com/nodeshift-starters/devfile-sample/blob/main/Dockerfile
```

Comment:
I'm not sure the Dockerfile belongs in this directory (`samples/devfiles`). Perhaps we can move it into a subdirectory?